MorphicServer/RegisterEndpoint.cs

```diff
@@ -197,9 +199,17 @@ public class BadRequestResponse
         [JsonPropertyName("error")] 
         public string Error { get; set; }
 
+        [JsonPropertyName("details")]
+        public Dictionary<string, int>? Details { get; set; }
```

Comment:
Lets make this Dictionary<string, object?>?  To match the signature Preferences uses for arbitrary json.

Reply:
Thanks. I knew there had to be a way to do this.